.eslintrc.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
         'import/no-unresolved': 'off', // Allows JS files to import TS files
         'import/prefer-default-export': 'off',
         'prefer-destructuring': ['error', { object: true, array: false }],
+        'no-unused-expressions': 'off', // note you must disable the base rule as it can report incorrect errors
```

Comment:
Hm, I don't know if this is worth the risk. Could we move these to the `**/__tests__/*` override section below?